doc/rules/basic/single_line_empty_body.rst

```diff
@@ -6,12 +6,26 @@ Empty body of class, interface, trait, enum or function must be abbreviated as
 ``{}`` and placed on the same line as the previous symbol, separated by a single
 space.
 
+Configuration
+-------------
+
+``only_for_constructors``
```

Comment:
Hi and thanks for your request.

Is this part of a standard or used by a bigger (F)OSS community? For us that is a condition to add a rule to this project. Otherwise, you can create a custom fixer and use that in your projects. Thanks for understanding.

Reply:
Personally I don't mind if this is part of a bigger community's standard. I just don't like the approach. IMHO it should be an option for configuring places where `{}` should be applied (constructor, method signature, loop, try/catch etc), by default all of them should be enabled, with ability to narrow the scope. This particular option is not flexible enough.

Reply:
I'm challenging the need for proposed option as well as for generic option.
I do not want to have option for sth because "it is technically possible", but because it's a recommended standard.

Until proven differently (with link to example community that will benefit from given configuration), I'm against config for this rule.

Reply:
It's not a standard as far as I know. It's just how I like it.

Reply:
@keradus it was discussed in #7809, it really looks like making it configurable is something nice to have.